Makefile

```diff
@@ -70,38 +70,69 @@ start_build:
 $(SITEBUILDDIR)/%.svg: $(SOURCEDIR)/%.svg
 	$(SCOUR) $(SCOUR_FLAGS) $< $@
 
-build: clean start_build build_images build_spec sphinx-build-docs
+build: clean start_build build_images build_spec build_docs
 
 build_images: $(SITEBUILDDIR)/$(OBJECTS)
 
 build_spec: build_spec_rfc build_spec_w3c
 
+# PS and RTF support are provided using a combination of wkhtmltopdf, pdf2ps and/or GNU enscript.
+requirements_xml2rfc:
+	sudo apt install xml2rfc enscript
+	pip install 'pycairo>=1.18' 'weasyprint<=0.42.3'
+
 # https://github.com/mmarkdown/mmark
 # sudo apt install -y mmark
 requirements_mmark:
 	wget https://github.com/mmarkdown/mmark/releases/download/v2.2.25/mmark_2.2.25_linux_amd64.tgz
 	tar xvzf mmark_2.2.25_linux_amd64.tgz
 	rm -f ./mmark*.tgz
+	sudo cp ./mmark /usr/local/bin/
```

Comment:
Any special reason to run it with sudo? I think we can also move this to gh-action steps. So any user can just install it on own OS. So we need just to check for it.